Benqi-Agents/DelegationMonitor/src/agent.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  Finding,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+  getEthersProvider,
+  ethers,
+  HandleTransaction,
+} from "forta-agent";
+import { BigNumber, providers, utils } from "ethers";
+import LRU from "lru-cache";
+import util from "./utils";
+
+const QI_CONTRACT: string = "0xf5841E5df027ed086A749Ecc33BFE7e0Df02B06F";
```

Comment:
The contract address is wrong, please update with with the correct QI token contract.
 For the PoC contract, you can save its address under another variable (`TEST_QI_CONTRACT` for example), I'd prefer not doing this in the agent file.